src/test/java/com/monumental/services/MonumentServiceTests.java

```diff
@@ -155,11 +156,11 @@ public void unitTestMonumentServiceGetAll_ListPassed() {
     }
 
     /**
-     * Test method for unit testing MonumentService.getAll(null)
+     * Test method for unit testing MonumentService.getAll()
      * Mocks the appropriate classes as to not connect to the database
      */
     @Test
-    public void unitTestMonumentServiceGetAll_NullPassed() {
+    public void unitTestMonumentServiceGetAll() {
         List<Monument> results = this.monumentService.getAll();
```

Comment:
Wasn't `MonumentService.getAll()` removed and replaced with `MonumentService.get(null)`?